rpcs3/Emu/NP/np_helpers.cpp

```diff
@@ -62,6 +62,13 @@ namespace np
 		// npid->reserved[0] = 1;
 	}
 
+	void string_to_country_code(std::string str, SceNpCountryCode& country_code)
```

Comment:
```suggestion
	void string_to_country_code(std::string_view str, SceNpCountryCode& country_code)
```

Avoids a redundant string copy. You'll have to change `.c_str()` to `.data()` below.